Maple2.Server.Game/PacketHandlers/MapleopolyHandler.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Maple2.Database.Storage;
+using Maple2.Model.Enum;
+using Maple2.Model.Error;
+using Maple2.Model.Game;
+using Maple2.Model.Game.Event;
+using Maple2.Model.Metadata;
+using Maple2.PacketLib.Tools;
+using Maple2.Server.Core.Constants;
+using Maple2.Server.Core.PacketHandlers;
+using Maple2.Server.Game.Packets;
+using Maple2.Server.Game.Session;
+
+namespace Maple2.Server.Game.PacketHandlers;
+
+public class MapleopolyHandler : PacketHandler<GameSession> {
+    public override RecvOp OpCode => RecvOp.Mapleopoly;
+
+    #region Autofac Autowired
+    // ReSharper disable MemberCanBePrivate.Global
+    public required ItemMetadataStorage ItemMetadata { private get; init; }
+    public required GameStorage GameStorage { private get; init; }
+    // ReSharper restore All
+    #endregion
+
+    private enum Command : byte {
+        Load = 0,
+        Roll = 1,
+        Result = 3,
+    }
+
+    public override void Handle(GameSession session, IByteReader packet) {
+        var function = packet.Read<Command>();
+        switch (function) {
+            case Command.Load:
+                HandleLoad(session);
+                return;
+            case Command.Roll:
+                HandleRoll(session);
+                return;
+            case Command.Result:
+                HandleResult(session);
+                return;
+        }
+    }
+
+    private void HandleLoad(GameSession session) {
+        using GameStorage.Request db = GameStorage.Context();
+        GameEvent? gameEvent = db.FindEvent(nameof(BlueMarble));
+        if (gameEvent?.EventInfo is not BlueMarble blueMarble) {
+            // TODO: Find error to state event is not active
+            return;
+        }
+
+        blueMarble.Tiles = blueMarble.Tiles.OrderBy(tile => tile.Position).ToList();
+
+        int.TryParse(session.GameEventUserValue.Get(GameEventUserValueType.MapleopolyFreeRollAmount, gameEvent).Value, out int freeRollValue);
```

Comment:
This is kinda ugly, what do you think about adding `.Int()`/`.String()` functions on `GameEventUserValueManager`. Open to other ideas too. I also don't like the fact that you need to pass `gameEvent` just for the scenario that the value needs to be instantiated, but can't think of a better structure right now.

Example:
```cs
int freeRollValue = session.GameEventUserValue.Get(GameEventUserValueType.MapleopolyFreeRollAmount, gameEvent).Int();
```